README.md

```diff
@@ -16,7 +16,15 @@ about more important problems than building a file uploader.
 **[Read the docs](https://uppy.io/docs)** |
 **[Try Uppy](https://uppy.io/examples/dashboard/)**
 
-<a href="https://transloadit.com" target="_blank"><img width="185" src="https://github.com/transloadit/uppy/raw/main/assets/developed-by-transloadit.png"></a>
+<p>
+  <a href="https://transloadit.com" target="_blank">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://github.com/transloadit/uppy/assets/375537/6651e57e-cb57-4336-8745-6473ae68d0bd">
+      <source media="(prefers-color-scheme: light)" srcset="https://github.com/transloadit/uppy/assets/375537/7f14421d-1e37-464e-8203-ade121216c88">
```

Comment:
nit: this doesn't seem necessary
```suggestion
```

Reply:
Thanks, Antoine! It removes extra repetition and I think it should work fine too, however GitHub Docs (unlike their blog article) have [a snippet with an explicit `prefers-color-scheme: light`](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to). I kept it for extra safety (in case there are any edge cases), but if you feel strongly, feel free to commit this suggestion.

Reply:
I don't know if it's a universal thing, I put `nit:` in my comment to clarify it's a non-blocking comment I don't feel strongly about. I think it originates from https://en.wikipedia.org/wiki/Nitpicking.